store/src/java/com/zimbra/cs/account/ProvUtil.java

```diff
@@ -1623,13 +1629,41 @@ private boolean execute(String args[]) throws ServiceException, ArgException, IO
             break;
         case MODIFY_HAB_GROUP_SENIORITY:
             modifyHabGroupSeniority(args);
+        case ADD_HAB_GROUP_MEMBER:
+            doAddMember(args);
+            break;
+        case DELETE_HAB_GROUP:
+            doDeleteDistributionList(args);
+            break;
+        case REMOVE_HAB_GROUP_MEMBER:
+            doRemoveMember(args);
+            break;
+        case GET_HAB_GROUP_MEMBERS:
+            doGetHABGroupMembers(args);
             break;
         default:
             return false;
         }
         return true;
     }
 
+    private void doAddMember(String[] args) throws ServiceException {
+        String[] members = new String[args.length - 2];
+        System.arraycopy(args, 2, members, 0, args.length - 2);
+        prov.addGroupMembers(lookupGroup(args[1]), members);
```

Comment:
do we support it both for soap and ldap rov.

Reply:
ldap and soap both are supported.